src/environments/environment.ts

```diff
@@ -4,7 +4,8 @@
 
 export const environment = {
   production: false,
-  hostname: 'http://localhost:3000'
+  hostname: 'http://localhost:3000',
+  url: 'http://localhost:3000/hero/',
```

Comment:
where is this used? this is way too specific

Reply:
it was used in the hero-service.ts. I deleted the url and solved the issue